### DISCUSSION TITLE
Why not include STIG gpo ?

### DISCUSSION THREAD

**Discussion by nliaudat** [LoicVeirman/HardenAD#135] [Ideas]
Hi, 
It could be great to extend the gpo from STIG source. 
It's available at https://public.cyber.mil/stigs/gpo/ 
Regards

**Comment by LoicVeirman** [comment]
Hi Nicolas,

Thanks for sharing ideas. STIGS is primarily focused on DoD if I didn't mistake myself, hence it will be a very hard stuff for anyone to implement all those challenging GPO (around 500 if I remember well).

The product is mainly intended to be applicable and manageable to company of any size - so the whole GPO package won't be a good idea. Some of them however should be relevant (or let's say even mandatory) to all.

If you have more precise idea on which GPO is missing, I'll be happy to discuss this with you here.

**Comment by nliaudat** [comment]
I use the dod gpo for many years and it need only small adjustments like removing some specifics parts of US department. 
Here you'll find the actual list : https://public.cyber.mil/announcement/gpo-update-9/

There is not a lot of GPO and many are workstations related (M365 and browsers).
The workstations gpo can be applied as is and could be imported perhaps as option

The interesting part is windows server GPO which are only 1 GPO per server type. 

I like the way you separate each security concern in a dedicated GPO and AD group, but it not so easy to maintain as a single GPO for "win2016 member server".

The counterside is they need a migtable and there is no changelog between version. 

In my opinion, the workstation part has not to be neglated as it could conduct to a full AD takeover

**Reply by LoicVeirman** [reply]
You're fully right about workstation (and I would have the same concern about servers as the admin generation tend to use them the same way they do with a workstation). We will add a new set of GPO in the future release (9.10.0 or 9.10.1, not yet pretty sure) which may already cover the same boundary: we'll get a look on them next. 

About the migTable, it's not a big deal: we already import the GPO this way with HAD ;) We use the classical *.migtable* file and we have added a special *.preftable* to work with the preference section. 

Another question is about the scope in concern through HardenAD: when hardening AD and its protocol, we know we are in the right place. Servers is enough mastered and standardized to make some critical change and avoid AD security risks by the side. Workstations, however, deeply involve the way users works... And that is a totally new challenge. So, a lot to think about ;)

Thnaks again for your input, I greatly appreciate and hope we'll get some other ones ;)

**Reply by nliaudat** [reply]
For workstations, the PAWS must be hardened and STIG GPO can do it in one shot.

**Comment by LoicVeirman** [comment]
That is absolutely true, which is embedded in the current design – we absolutely could onboard those GPO to the PAW Tier.

De : Nicolas Liaudat ***@***.***>
Date : jeudi, 19 septembre 2024 à 18:44
À : LoicVeirman/HardenAD ***@***.***>
Cc : Github ***@***.***>, Comment ***@***.***>
Objet : Re: [LoicVeirman/HardenAD] Why not include STIG gpo ? (Discussion #135)

CAUTION: This email originated from outside your organization. Exercise caution when opening attachments or clicking links, especially from unknown senders.

For workstations, the PAWS must be hardened and STIG GPO can do it in one shot.
